website/docs/audit.md

```diff
@@ -110,8 +110,9 @@ All of these events (including `violation_audited`) are marked with the same `au
 ## Configuring Audit
 
 - Audit violations per constraint: set `--constraint-violations-limit=123` (defaults to `20`)
-- Audit chunk size: set `--audit-chunk-size=500` (defaults to `0` = infinite) to limit memory consumption of the auditing `Pod`
+- Audit chunk size: set `--audit-chunk-size=400` (defaults to `500`, `0` = infinite) Lower chunk size can reduce memory consumption of the auditing `Pod` but can increase the number requests to the Kubernetes API server.
 - Audit interval: set `--audit-interval=123` (defaults to every `60` seconds). Disable audit interval by setting `--audit-interval=0`
+- Audit write to RAM disk (Gatekeeper v3.7.0+): By default, audit writes to the disk attached to the node. If there are concerns with high IOPS, then use helm install `--set audit.writeToRAMDisk=true` to allow audit to write to a tmpfs RAM disk instead.
```

Comment:
can we mention tradeoff for memory if using ramdisk?

Reply:
Also, add a mention of modifying the manifest to mount a ramdisk if not using Helm?